docker/scripts/osie.sh

```diff
@@ -95,9 +95,14 @@ else
 	fi
 	early_phone=1
 	custom_image=true
+
+	# Re-DHCP so we obtain an IP that will last beyond the early phone_home
+	reacquire_dhcp "$(ip_choose_if)"
```

Comment:
I think this should be down below when we act on the `early_phone -eq 1` case. Lets keep all the action bits together.

Reply:
That's reasonable. I've updated the PR with that change.